frontend/src/components/header/spaceHeader/spaceManagerSpaceHeader/SpaceManagerSpaceHeader.tsx

```diff
@@ -0,0 +1,60 @@
+import type { SpaceAccessType } from '../../../../types/space.type';
+import type { Timer } from '../../../../types/timer.type';
+import IconLabelButton from '../../../@common/buttons/iconLabelButton/IconLabelButton';
+import SpaceHeader from '../SpaceHeader';
+
+interface IconItem {
+  /** 아이콘 요소    */
```

Comment:
```suggestion
  /** 아이콘 요소 */
```

스페이스(!)가 여러번 입력됐네요!
스페이스 매니저 스페이스 헤더의 과도한 스페이스 이슈...😇